controller/server/view/network_details_page.ml

```diff
@@ -225,8 +230,23 @@ let connected_form service =
         ]
     ]
 
+let unsupported_notice service =
+    p ~a: [ a_class ["d-Note"] ][
+      txt "Connecting to this network is not possible, because it uses an unsupported authentication protocol."
+    ; br ()
+    ; txt @@ Printf.sprintf "Available authentication protocols for this network: %s"
+        (String.concat ", " (List.map
+            (fun s -> Sexplib.Sexp.to_string (sexp_of_security s))
+            service.security))
+    ]
+
 let html service =
   let is_service_connected = Connman.Service.is_connected service in
+  let is_service_supported =
+      List.exists (fun e -> List.mem e service.security) [ PSK; WEP; None ]
+    ||
+      (service.security = []) (* wired connections have this *)
```

Comment:
Strictly speaking this extra case is not needed, because currently `is_service_supported` is only relevant for wifi connections, but I added it to make the code more future proof in case this changes somehow.

Reply:
Seems reasonable :+1: